pkg/cmd/release/deploy/deploy.go

```diff
@@ -341,7 +341,7 @@ func deployRun(cmd *cobra.Command, f factory.Factory, flags *DeployFlags) error
 
 // scheduledStartTimeAnswerFormatter is passed to the DatePicker so that if the user selects a time within the next
 // one minute after 'now', it will show the answer as the string "Now" rather than the actual datetime string
-func scheduledStartTimeAnswerFormatter(datePicker *surveyext.DatePicker, t time.Time) string {
+func ScheduledStartTimeAnswerFormatter(datePicker *surveyext.DatePicker, t time.Time) string {
```

Comment:
this (along with various other things in this file) is now public so that `runbook run` can re-use it